setup.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+import os
+from os.path import join, dirname, abspath
+from pkgutil import extend_path
+from setuptools import find_packages, setup
+import shutil
+
+with open(join(os.path.dirname(__file__), 'README.md')) as readme:
+    README = readme.read()
+
+# Allow setup.py to be run from any path
+os.chdir(os.path.normpath(join(abspath(__file__), os.pardir)))
+
+
+def load_requirements():
+    return open(join(dirname(__file__), 'requirements-pkg.txt')).readlines()
```

Comment:
Why don't we install all the requirements?

Reply:
Because for the files that we need (models and some util) we don't require so many dependencies. The less, the better.